test/bdd/steps/api/search-assertion.js

```diff
@@ -0,0 +1,76 @@
+const { When, Then, Given } = require('@cucumber/cucumber');
+const { expect, assert } = require('chai');
+const utilities = require('../../../utilities/utilities');
+let handlerId, searchResult, searchData, parsedKeywords, publishData;
```

Comment:
Don't define constants here, use state for all data that needs to be passed between test steps